spec/requests/ai_helper/assistant_controller_spec.rb

```diff
@@ -107,4 +107,47 @@
       end
     end
   end
+
+  describe "#caption_image" do
+    let(:image_url) { "https://example.com/image.jpg" }
+    let(:caption) { "A picture of a cat sitting on a table" }
+
+    context "when logged in as an allowed user" do
+      fab!(:user)
+
+      before do
+        sign_in(user)
+        user.group_ids = [Group::AUTO_GROUPS[:trust_level_1]]
+        SiteSetting.ai_helper_allowed_groups = Group::AUTO_GROUPS[:trust_level_1]
+        SiteSetting.ai_helper_model = "open_ai:gpt-4-vision-preview"
```

Comment:
You can set the model to `fake:fake` to bypass validations. We should probably bypass them when running tests. It's a bit annoying

```suggestion
        SiteSetting.ai_helper_model = "fake:fake"
```